packages/create/templates/hello-v2/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "dbos-hello-koa",
```

Comment:
Do we want to call this template `hello-koa`? We don't use the word `v2` anywhere else.

Reply:
It was supposed to be v2.  We do have a longer-term plan to make this into the main hello when it is ready.
I do not want to call it hello-koa.  If you want koa, use our nice serverless stuff that helps you get everything right.